licensing-and-activation/activation-faq.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Activation FAQ
+description: >-
+  This article provides answers to most frequently asked questions regarding
+  activation.
+---
+
+# Activation FAQ
+
+## What features are available in the trial?
+
+The trial is the latest released application version containing all available application features.
+
+## What are the trial limitations?
+
+The trial version of SysKit Trace has a couple of limitations:
+
+* Limited information in the generated documentation and exported reports. Export to .docx or .xlsx formats is not available. You can see examples of how exported [reports](https://www.syskit.com/products/trace/resources/documentation-report-examples/) and [documentation](https://www.syskit.com/products/trace/resources/documentation-report-examples/) look like in the full version.
```

Comment:
Are we sure we are going to have these links? Also maybe it would be good to have them as app links as well?

Reply:
these are not avaliable on the current web, there is no resources page, dora said that the rest of the generated documentation except the xlsx file was ugly. we can either add the page with the examples or I can remove these references here. I vote for removal of the text for now and a PBI for later

Reply:
I agree

Reply:
i'll mark this as resolved.